_sass/elements/_links.scss

```diff
@@ -28,20 +33,40 @@ a,
   &.button-secondary {
     border: none;
   }
-  img {
+  img, i, span {
     border: none;
   }
 
 }
 .link-beta {
-  @extend a;
+  @extend %link;
   color: $white;
   &:active,
   &:hover {
     text-decoration: none;
-    border-bottom: 1px dotted $white;
-    img{
+    border-bottom: $thin-border-size dotted $white;
+    img, i, span{
```

Comment:
Rule declaration should be preceded by an empty line<br>Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space